### ISSUE TITLE
Click & Collect not reserving items

### ISSUE THREAD

**Issue by muety** [leih-lokal/scripts#19]
We keep having the issue that items reserved via click & collect don't appear on our reservation list and remain "in stock" in our catalog.

Example:
1. Reservation for `0002` (user `2869`) on Dec 8th (11:46 pm)
1. Reservation for `3602, 1302` (user `630`) on Dec 8th (12:38 pm)
1. Reservation for `3021, 3415, 1035` (unregistered user) on Dec 18th (2:14 pm)

For (1) and (2), the reservation couldn't be auto-accepted, because items were allegedly out of stock, see [logs](https://github.com/leih-lokal/scripts/actions/runs/7146311678/job/19463742710), and an _"Bitte ansehen"_ mail had been sent. However, I don't see any rentals for neither of the items on that day.

For (3), the reservation was auto-accepted, status was changed properly, but then changed back to `instock` within the same pass of the script see [logs](https://github.com/leih-lokal/scripts/actions/runs/7146311678/job/19463742710).

Needs investigation!

**Comment by muety** [comment]
The appointment manager webhook seems to have not been getting triggered anymore recently. Might be related to the fact that we kicked the @leihlokal user from the org (probably that user's access token was used). Opted in to [fine-grained access tokens](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-programmatic-access-to-your-organization/managing-requests-for-personal-access-tokens-in-your-organization#about-fine-grained-personal-access-token-requests) now, created one for myself and permitted its use for this repo. Adapted the WordPress trigger script accordingly.

Regarding my above observations:
* Re. (1) and (2): as the appointment manager script ran on schedule in the evening, all items had already been picked up by the customers during the day (even though not reserved), so of course they weren't available anymore then.
* Re. (3): apparently the customer didn't pick up their items on that day (no according rentals in our system), so the appointment manager did its job and properly changing them back to available status.

**TL;DR:** GitHub access token had become invalid. Creating a new one fixed the issue.

I do remember that we used to have wrong statuses for otherwise properly reserved items in the past, though. That's probably a whole different issue, but let's wait until it pops up again.
